packages/react/src/components/form/form.test.tsx

```diff
@@ -25,6 +26,32 @@ describe("#Form", () => {
     );
 
     expect(screen.getByTestId("sid-form-initial-state")).toBeInTheDocument();
+
+    expect(screen.queryByText(TEXT["initial.divider"])).not.toBeInTheDocument();
```

Comment:
@kaspermroz this is **not** my domain, so please put my feedback in context :)
my two cents: imho this test assumes the initial state doesn't have OIDC factors when it shouldn't => I think it'd be more correct to keep this `should render in the initial state` test as-is and add a separate one specific to rendering without the divider where you explicitly don't have OIDC factors.